docs/user-guide/nextcloud-integration/README.md

```diff
@@ -120,12 +119,36 @@ When a Nextcloud file or folder is linked to a work package, an OpenProject user
 
 However, all available actions depend on permissions the OpenProject user (or more precisely, the Nextcloud account tied to that user) has in Nextcloud. In other words, a user who does not have the permission to access the file in Nextcloud will also *not* be able to open, download, modify or unlink the file in OpenProject.
 
+## Disconnecting your Nextcloud and OpenProject accounts
+
+In certain cases, you may wish to disconnect your Nextcloud and OpenProject accounts. You might want to do this, for example, if you have multiple accounts and realise that you connected the wrong ones. 
+
+Since OAuth credentials were generated by both Nextcloud and OpenProject when you connected your accounts, we recommend disconnecting on both ends.
```

Comment:
I don't think both steps are always necessary. But I see that it might be hard to find out which account is wrongly connected.

Reply:
We discussed this with @wielinde yesterday, but perhaps there are easier ways to communicate this. The thing is, if you only disconnect on one end, we'll have to explain to the user what happens to the end where they didn't disconnect. (Or indeed, that information can still flow from oen side).

We can discuss this this afternoon during our call.